### PR TITLE
SLIDESDOC-476 Parallel conversion of presentation slides into images

### PR DESCRIPTION
**Pull request by andreypotapovaspose**
Added keywords and the section "Convert Presentation Slides to Images in Parallel" to the technical article "Multithreading in Aspose.Slides" (.NET).

### CODE REVIEW

**Review by valentynlukianets-aspose — APPROVED** [review]
Approved for Python

File: php-java/developer-guide/technical-articles/multithreading/_index.md [hunk 1]
```diff
@@ -3,13 +3,22 @@ title: Multithreading in Aspose.Slides
...
 ---
 
-{{% alert color="primary" %}} 
+## **Introduction**
 
 While parallel work with presentations is possible (besides parsing/loading/cloning) and everything goes well (most times), there is a small chance you might get incorrect results when you use the library in multiple threads.
 
 We strongly recommend that you do **not** use a single [Presentation](https://reference.aspose.com/slides/php-java/aspose.slides/Presentation) instance in a multi-threading environment because it might result in unpredictable errors or failures that are not easily detected.
 
 It is **not** safe to load, save, and/or clone an instance of a [Presentation](https://reference.aspose.com/slides/php-java/aspose.slides/Presentation) class in multiple threads. Such operations are **not** supported.  If you need to perform such tasks, you have to parallel the operations using several single-threaded processes—and each of these processes should use its own presentation instance.
```
Comment (petrtetenko) [inline php-java/developer-guide/technical-articles/multithreading/_index.md:24]: @andreypotapovaspose 
Is this how it was intended or is there some other reason why there is no example for PHP?

**Review by petrtetenko — CHANGES_REQUESTED** [review]

File: php-java/developer-guide/technical-articles/multithreading/_index.md (hunk 1 above)
Comment (andreypotapovaspose) [reply php-java/developer-guide/technical-articles/multithreading/_index.md:24]: @petrtetenko,
PHP does not have built-in support for multithreading. It is necessary to use a third-party library to write such an example.

Comment (petrtetenko) [reply php-java/developer-guide/technical-articles/multithreading/_index.md:24]: @andreypotapovaspose 

Perhaps this should be indicated separately in the text?

Comment (andreypotapovaspose) [reply php-java/developer-guide/technical-articles/multithreading/_index.md:24]: @petrtetenko,
Added the text.

**Review by petrtetenko — APPROVED** [review]

**Pull request merged**